### PR TITLE
Ngwmn 1041 implement support of dual attribution

### PR DESCRIPTION
**Pull request by abriggs-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

NGWMN 1041 and 1698
-----------
Refines the provider logos and implements filtering of SIFTA calls by current water year 

Description
-----------
The biggest changes here are addition of a call to the SIFTA service to grab the cooperator logos. We were using an in-application process for adding logos for the non-usgs locations. This is still used in addition to the call to SIFTA. The SIFTA call is filtered to only allow providers from the current water year. The results are further filtered to only add one USGS logo, no matter how many exist and to only use one type of USGS logo and link.

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: server/ngwmn/filters.py [hunk 1]
```diff
@@ -20,3 +20,17 @@ def asset_url_filter(asset_src):
     manifest = app.config.get('ASSET_MANIFEST')
     asset_path = manifest[asset_src] if manifest else asset_src
     return urljoin(app.config.get('STATIC_ROOT'), asset_path)
+
+
+@app.template_filter('https_url')
+def https_url(url):
```
Comment (mbucknell) [inline server/ngwmn/filters.py:26]: I'd like to see some tests for this.

File: server/ngwmn/templates/site_location.html [hunk 2]
```diff
@@ -269,13 +269,67 @@ <h1>{{ feature.SITE_NAME }}</h1>
...
+    {% endif %}
+    {% if cooperators %}
+        {% for cooperator in cooperators | sort(attribute='Name') %}
+            {% if not 'usgs' in cooperator.URL %}
+            <figure class="provider-logo">
+                <a class="usa-class"
+                   href="{{ cooperator.URL }}"
+                   target="_blank"
+                   rel="noopener"
+                >
```
Comment (mbucknell) [inline server/ngwmn/templates/site_location.html:297]: Same as above. Maybe put on line 296.

File: server/ngwmn/templates/site_location.html [hunk 3]
```diff
@@ -269,13 +269,67 @@ <h1>{{ feature.SITE_NAME }}</h1>
...
+                    <img src="{{ cooperator.IconURL | https_url }}"
+                         alt="logo for {{ cooperator.Name }}"
+                         height="50">
+                </a>
+                <figcaption>
+                    <a  class="usa-class"
+                        href="{{ cooperator.URL }}"
+                        target="_blank"
+                        rel="noopener"
+                    >
```
Comment (mbucknell) [inline server/ngwmn/templates/site_location.html:307]: Ditto.

**Review by mbucknell — CHANGES_REQUESTED** [review]
Some nits and I'd like to see some tests for the new template filter.

File: server/ngwmn/templates/site_location.html (hunk 2 above)
Comment (abriggs-usgs) [reply server/ngwmn/templates/site_location.html]: fixed

File: server/ngwmn/templates/site_location.html (hunk 3 above)
Comment (abriggs-usgs) [reply server/ngwmn/templates/site_location.html]: also fixed

**Review by mbucknell — APPROVED** [review]

**Pull request merged**